azure-pipelines.yml

```diff
@@ -192,6 +192,32 @@ jobs:
         env:
           GH_TOKEN: $(GH_TOKEN)
 
+  - job: npm_beta_release
+    displayName: 'NPM beta release'
+    condition: and(succeeded(), contains(variables['Build.SourceBranch'], 'refs/heads/master'))
+    dependsOn:
+      - lint
+      - test_broker
+      - test_admin
+      - test_producer
+      - test_consumer
+      - test_others
+    pool:
+      vmImage: 'Ubuntu 16.04'
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '8.x'
+      - bash: ./scripts/betaRelease.js
+        displayName: build beta release
+      - task: Npm@1
+        inputs:
+          command: custom
```

Comment:
Maybe we should tag with the new beta version so that you can find which commits are in the beta release. Would help to generate changelogs etc if we really want to as well.